### PR TITLE
ENH: Add Neuro4Neuro paper review

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add Neuro4Neuro paper review.

### CODE REVIEW

File: article/_posts/2020-06-04-Neuro4Neuro.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+The main advantages of the proposed method are:
+- Acceleration wrt to conventional methods.
+- Avoiding additional steps required by other methods, such as parcellation,
+atlas registration and fiber tracking.
+
+
+# Comments
+
+- The choice of the input providing the best results and other hyper-parameters
+where decided based on experiments done on the forceps minor. This looks a
```
Comment (AntoineTheb) [inline article/_posts/2020-06-04-Neuro4Neuro.md:99]: were decided

Comment (AntoineTheb) [inline article/_posts/2020-06-04-Neuro4Neuro.md:99]: This looks _like_ a limitation ?

Comment (jhlegarreta) [reply article/_posts/2020-06-04-Neuro4Neuro.md]: Thanks 👍 . Re-worded in 1c2e4b7.

Comment (jhlegarreta) [reply article/_posts/2020-06-04-Neuro4Neuro.md]: Thanks 👍 . Fixed in 1c2e4b7.

**Pull request merged**